src/main/java/com/troubleshooting/troubleshootingtool/ContainerDataRetrieval.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * Retrieves the data returned from the Docker Client and packs the data in the appropriate data models
+ */
+package com.troubleshooting.troubleshootingtool;
+
+import com.troubleshooting.model.ConjurLogModel;
+import com.troubleshooting.model.EnvironmentModel;
+import com.troubleshooting.model.EnvironmentsModel;
+import com.troubleshooting.model.LogsModel;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+@Component
+public final class ContainerDataRetrieval implements IDataRetrieval {
+    private ContainerDataAccess access;
+    private ConjurLogModel log;
```

Comment:
avoid abbreviating names

Reply:
Abbreviating being `log` instead of `conjurLogModel`?

Reply:
yes, and in general be explicit remember someone else will read your code